docs/Hooks.md

```diff
@@ -86,7 +116,7 @@ fastify.addHook('preValidation', async (request, reply) => {
   return
 })
 ```
-**Notice:** in the [preValidation](#preValidation) hook, `request.body` will always be `null`, because the body parsing happens before the [preValidation](#preHandler) hook.
+**Notice:** in the [preValidation](#preValidation) hook, `request.body` will always be `null`, because the body parsing happens before the [preHandler](#preHandler) hook.
```

Comment:
This sentence should be removed since it isn't true: it is the raw body without the JSON schema check applied

This print the body: 

```
  fastify.addHook('preValidation', function (request, reply, next) {
    console.log('preValidation', request.body)
    next()
  })
```

Reply:
Right. Will do.